java-tools/OsmAndMapCreatorUtilities/src/main/java/net/osmand/obf/preparation/IndexCreationContext.java

```diff
@@ -249,18 +249,20 @@ private static String serialize(TreeSet<String> lst) {
 		return bld.toString();
 	}
 
+	public boolean isInsideRegionBBox(double lat, double lon) {
```

Comment:
- Check specific tags like coastline (always exclude) - CC @xmd5a2 
- Check multipolygon countries like islands in Indonesia